recon/src/recon.rs

```diff
@@ -258,6 +258,10 @@ where
 
         Ok(new)
     }
+    /// Report all keys in the range that are missing a value
+    pub async fn keys_with_missing_values(&mut self, range: RangeOpen<K>) -> Result<Vec<K>> {
```

Comment:
out of curiosity, why implement `range_with_values` and `keys_with_missing_values` rather than just `range_with_opt_values` and figure both out in one a level above storage? Not sure if it's ergonomics, the list of NULLs being big, or something I'm missing.

Reply:
In the case of keys_with_missing_values I don't want the values. So if I had to read all key/values in a range just to discard the values that actually existed that would be wasteful disk io.